Source/Common/Config.cpp

```diff
@@ -338,10 +338,13 @@ fextl::string RecoverGuestProgramFilename(fextl::string Program, bool ExecFDInte
 }
 
 ApplicationNames LoadConfig(fextl::unique_ptr<FEX::ArgLoader::ArgLoader> ArgsLoader, bool LoadProgramConfig, char** const envp,
-                            bool ExecFDInterp, int ProgramFDFromEnv) {
-  FEX::Config::InitializeConfigs();
+                            bool ExecFDInterp, int ProgramFDFromEnv, const PortableInformation* PortableInfo) {
```

Comment:
We don't need pointer semantics here. `PortableInformation` is cheap enough to zero-construct (and is already non-null in most places anyway), so this can be a reference instead.

Reply:
Switched over to a reference.